apps/mobile/src/components/TabNavivation/Item.tsx

```diff
@@ -35,4 +46,13 @@ const styles = StyleSheet.create({
 	tabTitle: {
 		textAlign: 'right',
 	},
+	icon: {
+		borderRadius: 1000,
```

Comment:
don't over use radius value like this, it won't work on certain case, use `height / 2` instead.